release-notes.md

```diff
@@ -1,5 +1,25 @@
 # Release Notes
 
+## 3.8.0
+
+### Ability to edit a contact type
+
+It is now possible to edit a contact type. There is a new flag `--updateOfflineDocs` that can be passed to the `edit-contacts` action which updates docs in the directory provided in `--docDirectoryPath` instead of downloading the docs from the database.
```

Comment:
Please mention which action can be used to edit contact types. 
```suggestion
It is now possible to edit a contact type with the `edit-contacts` action. Additionally, `edit-contacts` accepts a new flag `--updateOfflineDocs`, that allows editing json docs that already exist in the directory provided in `--docDirectoryPath` instead of downloading the docs from the database.
```